pkgs/tools/security/pwgen_secure/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, python3Packages, fetchFromGitHub }:
+
+with python3Packages;
+
+buildPythonApplication rec {
+  pname = "pwgen_secure";
+  version = "0.9.1";
+
```

Comment:
We need to use the `disabled` argument to disable anything older than the documented python versions
* https://github.com/mjmunger/pwgen_secure/blob/master/setup.py#L19
* https://github.com/mjmunger/pwgen_secure#installation